src/services/ls-migration/addedSafes.ts

```diff
@@ -23,11 +23,25 @@ type OldAddedSafes = Record<
     address: string
     chainId: string
     ethBalance: string
-    owners: string[]
+    owners: Array<string | { name?: string; address: string }>
     threshold: number
   }
 >
 
+export const migrateAddedSafesOwners = (
+  owners: OldAddedSafes[string]['owners'],
+): AddedSafesState[string][string]['owners'] => {
+  return owners.map((value) => {
+    if (typeof value === 'string') {
+      return { value }
+    }
+    return {
+      value: value.address,
+      ...(value.name && { name: value.name }),
```

Comment:
You're assuming here that if it's not a string, it's necessarily `{ name?: string; address: string }`. I would add a type check here as well and not write anything if it's neither.

Reply:
I've adjusted it accordingly and added checksum verification to match that of the address book migration as well.